src/components/Store/StoreEdit.vue

```diff
@@ -216,6 +216,9 @@ export default {
         ],
       })
     },
+    cancel () {
```

Comment:
What's your reasoning putting it into the component vs putting it in the parent?

Reply:
There are 2 vue files (Create.vue and Edit.vue) that use the same StoreEdit.vue file. In stead of changing 2 files, I chose to edit just 1.
`yarn run v1.3.2
$ node test/jestUTC.js -u`
This is the output when I run `yarn test -u`

Reply:
> There are 2 vue files (Create.vue and Edit.vue) that use the same StoreEdit.vue file. In stead of changing 2 files, I chose to edit just 1.

That's a valid goal, but not necessarily the best when it comes to software architecture.

We're trying to approach the concept of [presentational components](https://medium.com/@dan_abramov/smart-and-dumb-components-7ca2f9a7c7d0) and we tried to split it so that presentational components go into `src/components/`, while container components go into `src/pages/`.

By that logic, I would say that routing decisions belong to the container component, because it's a state transition.

We're not perfect with the split, but I would say that `StoreEdit` is one of the more _pure_ presentational components in karrot and I would like to keep it that way.